eng/tools/generate-doc/index.ts

```diff
@@ -27,16 +27,22 @@ async function runTypeDoc({ outputFolder, cwd }: { outputFolder: string; cwd: st
   }
   console.log(`entrypoints: ${JSON.stringify(entryPoints)}`);
 
-  const app = await TypeDocApplication.bootstrap({
-    entryPoints,
-    excludeInternal: true,
-    excludePrivate: true,
-    skipErrorChecking: true,
-    theme: "azureSdk",
-  }, [
-    new TSConfigReader(),
-    new TypeDocReader(),
-  ]);
+  const srcTsconfigPath = path.join(cwd, "tsconfig.src.json");
+  const tsconfig = (await fileExists(srcTsconfigPath))
```

Comment:
nitpick: we probably should remove this helper and switch to use `fs.existsSync`